### PR TITLE
feat: localized messages

### PR DESCRIPTION
**Pull request by adlerhurst**

### CODE REVIEW

**Comment by adlerhurst** [comment]
@fforootd please review language files for spelling. (`internal/static/i18n/de.yaml`, `internal/static/i18n/en.yaml`)

File: internal/static/i18n/de.yaml [hunk 1]
```diff
@@ -113,4 +113,171 @@ Errors:
...
-    NotFound: UserSession konnte nicht gefunden werden
\ No newline at end of file
+    NotFound: UserSession konnte nicht gefunden werden
+EventTypes:
+  user:
+    added: Benutzer hinzugefügt
+    selfregistered: Benutzer hat sich selbst registriert
+    initialization: 
+      code:
+        added: Initialisierungscode hinzugefügt
```
Comment (fforootd) [inline internal/static/i18n/de.yaml:123]:
```suggestion
        added: Initialisierungscode generiert
```

i think `generiert` is the better word

File: internal/static/i18n/en.yaml [hunk 2]
```diff
@@ -113,4 +113,171 @@ Errors:
...
+    password:
+      complexity:
+        added: password complexity policy added
+        changed: password complexity policy changed
+      age:
+        added: password age policy added
+        changed: password age policy changed
+      lockout:
+        added: password lockout policy added
+        changed: password lockout policy changed
```
Comment (fforootd) [inline internal/static/i18n/en.yaml:267]:
```suggestion
        added: Password complexity policy added
        changed: Password complexity policy changed
      age:
        added: Password age policy added
        changed: Password age policy changed
      lockout:
        added: Password lockout policy added
        changed: Password lockout policy changed
```

File: internal/static/i18n/en.yaml [hunk 3]
```diff
@@ -113,4 +113,171 @@ Errors:
...
+        set: Global org set
+    project:
+      iam:
+        set: Zitadel project set
+    member:
+      added: Zitadel member added
+      changed: Zitadel member changed
+      removed: Zitadel member removed
+  key_pair:
+    added: key pair added
```
Comment (fforootd) [inline internal/static/i18n/en.yaml:283]:
```suggestion
      started: ZITADEL setup started
      done: ZITADEL setup done
    global:
      org:
        set: Global org set
    project:
      iam:
        set: ZITADEL project set
    member:
      added: ZITADEL member added
      changed: ZITADEL member changed
      removed: ZITADEL member removed
  key_pair:
    added: Key pair added
```

File: internal/static/i18n/de.yaml (hunk 1 above)
Comment (adlerhurst) [reply internal/static/i18n/de.yaml:123]: I agree

File: internal/static/i18n/de.yaml [hunk 4]
```diff
@@ -113,4 +113,171 @@ Errors:
...
+      changed: Verwaltungszugriff geändert
+      removed: Verwaltungszugriff entfernt
+      deactivated: Verwaltungszugriff deaktiviert
+      reactivated: Verwaltungszugriff reaktiviert
+      cascade:
+        changed: Verwaltungszugriff geändert
+      member:
+        added: Verwaltungszugriffsmitglied hinzugefügt
+        changed: Verwaltungszugriffsmitglied geändert
+        removed: Verwaltungszugriffsmitglied entfernt
```
[the remark this answers is not in the document]
Comment (fforootd) [reply internal/static/i18n/de.yaml:241]: i have no better idea right now, but we should replace this in the future

**Pull request merged**